### ISSUE TITLE
[Bug] Unable to display demo details of the components from angular library

### ISSUE THREAD

**Issue by akhileshThapliyal** [ng-doc/ng-doc#168]
### Is this a regression?

Yes

### Description

I am creating an angular library which I intend to import into my ng doc application. I want to refer to the components which are in my library so that I can maintain different documentation versions and in my library, I will have different versions of the components. For instance, I intend to have v1/docs, v2/docs applications and an angular library with v1 and v2 components.

Current Behaviour: When I try to load a component in the demo panel of the ng doc, it does render the component but doesn't show up the source code when the show code is toggled. This doesn't happen for components which are local to the application.

Expected Behavior: I should be able to view the demo panel with the source code of the components imported from an angular library.

Github Repo: https://github.com/akhileshThapliyal/my-ng-doc-app

ng-doc-tag/ng-doc.page.ts

```typescript
import {NgDocPage} from '@ng-doc/core';
import { NgDocLabelComponent, NgDocTagComponent } from '@ng-doc/ui-kit';
import { ButtonComponent } from '../button/button.component';

const NgDocTagPage: NgDocPage = {
	title: `Ng Doc Tag`,
	mdFile: './index.md',
	demos: { 
		NgDocLabelComponent
	 },
	imports: [
		NgDocLabelComponent
	]
};

export default NgDocTagPage;
``` 

ng-doc-tag/index.md

```typescript
{{ NgDocActions.demo("NgDocLabelComponent", {inputs: {"ng-doc-label": "Defaul Ng Doc Label"} }) }}
``` 

<img width="868" alt="image" src="https://github.com/ng-doc/ng-doc/assets/7280745/337abb76-9a2b-450e-9374-520907f026ac">

<img width="854" alt="image" src="https://github.com/ng-doc/ng-doc/assets/7280745/dfd6dc53-7e72-4b9d-adfa-858ef2e2fd97">

### OS

Unix (Linux, macOS, etc.)

### Browser

Chrome

### Node version

20.9.0

### Anything else?

Ng Doc version: 17.6.10

**Comment by skoropadas** [comment]
Thanks, I'll check the repo

**Comment by skoropadas** [comment]
I have checked the repository, it is not a bug but expected behavior. I am not quite sure what code you want to see by pointing to a component of an already compiled library.

Demos are meant to show an example of how to use a particular component, so you should create a new demo component in which you demonstrate an example of using your component.

I also noticed that you are adding components in the import field. This will not cause any errors, but it is not necessary if you are using standalone components.

**Comment by akhileshThapliyal** [comment]
Hi @skoropadas, I managed it. So I created an angular workspace where I have all the libs and projects co-exist. I am referencing using a relative path to the components in the libs. Not sure whether this is the optimal approach, but it helps me to keep the code clean, tidy and organised. The source codes are now parsed and shown in the markdown file.

**_NgDoc is such an easy, flexible, beautiful and powerful documentation tool for angular. As this is built using Angular, it is super easy to customize things which isn't the case with other available documentation tools._**
